Dockerfile.el7

```
@@ -15,7 +15,7 @@ RUN yum install -y yum-utils \
     && yumdownloader \
         --destdir . \
         --source \
-        --exclude amazon-linux-extras,amazon-linux-extras-yum-plugin,amazon-linux-onprem,amazon-ssm-agent,amazonlinux-indexhtml,aws-cfn-bootstrap,aws-cli-plugin-cloudwatch-logs,awscli,awslogs,ec2sys-autotune \
+        --exclude amazon-linux-extras,amazon-linux-extras-yum-plugin,amazon-linux-onprem,amazon-ssm-agent,amazonlinux-indexhtml,aws-cli-plugin-cloudwatch-logs,awscli,awslogs,ec2sys-autotune \
         amazon\* aws\* ec2\*
```

Something crazy to try...

```suggestion
        --exclude amazon-linux-extras,amazon-linux-extras-yum-plugin,amazon-linux-onprem,amazon-ssm-agent,amazonlinux-indexhtml,aws-cli-plugin-cloudwatch-logs,awscli,awslogs,ec2sys-autotune \
        amazon\* aws\* ec2\* python3-daemon python3-pystache
```

Perhaps we can build those from source?

---

Ok.  🤞

---

Well that didn't work cleanly. Would need to do some more investigation to figure out what's up. Alternatively, we could acknowledge that the AWS docs recommend using the tar.gz package for platforms other than Amazon Linux, and figure out how to get that working in the amigen projects.